tests/robot-tests/tests/admin_and_public/bau/publish_release_and_amend_2.robot

```diff
@@ -373,10 +373,13 @@ Edit methodology
     user clicks link  manage methodologies
     user clicks element  //*[@id="approved-methodologies-tab"]
     user clicks link  ${METHODOLOGY_NAME}
+    user clicks link  Sign off
```

Comment:
With these new additions to this test case, I've split this into three separate test cases instead - just so if there is a failure when changing the methodology status to Draft or Approved it's a bit clearer when looking at the test logs